packages/astro/test/lit-element.test.js

```diff
@@ -5,6 +5,7 @@ import { loadFixture } from './test-utils.js';
 let fixture;
 
 const NODE_VERSION = parseFloat(process.versions.node);
+const LIT_PART = (content) => `<!--lit-part-->${content}<!--/lit-part-->`;
```

Comment:
This is what we use in lit-html tests: https://github.com/lit/lit/blob/main/packages/lit-html/src/test/test-utils/strip-markers.ts

Reply:
Those regexes don't match the output here. Do you know why this would be rendering like so? Otherwise adding a regex would likely just add complexity

Reply:
went with a simple regex that only strips `lit-part`. Simple enough for now. Can include our "official" regexes later if needed. I don't want to add the complexity now.